app/src/androidTest/java/org/simple/clinic/patient/PatientRepositoryAndroidTest.kt

```diff
@@ -1425,11 +1425,12 @@ class PatientRepositoryAndroidTest {
 
       appointmentDate?.let {
         appointmentRepository.schedule(
-            appointmentUuid = UUID.fromString("cf002164-3032-4a91-8dbe-15c0b029db51"),
             patientUuid = patientUuid,
+            appointmentUuid = UUID.fromString("cf002164-3032-4a91-8dbe-15c0b029db51"),
             appointmentDate = it,
             appointmentType = Manual,
-            facilityUuid = testData.qaFacility().uuid
+            appointmentFacilityUuid = testData.qaFacility().uuid,
```

Comment:
We have a variable called `currentFacility` that overrides the getter to return `testData.qaFacility()` in `PatientRepositoryAndroidTest` class. Can we use that instead of calling `testData.qaFacility()` in multiple places?

Reply:
Done